quora-api/src/main/java/com/upgrad/quora/api/controller/UserController.java

```diff
@@ -74,7 +74,7 @@
         String[] splitText = authorization.split("Basic ");
         SigninResponse signinResponse = null;
         HttpHeaders headers = new HttpHeaders();
-        byte[] decoder = Base64.getDecoder().decode(splitText[0]);
+        byte[] decoder = Base64.getDecoder().decode(splitText[1]);
```

Comment:
please merge your code, i have already checked in the code

Reply:
Sure.